vm/hv1/hv1_emulator/src/cpuid.rs

```diff
@@ -57,6 +57,11 @@ pub fn hv_cpuid_leaves(
             //     .with_fast_hypercall_output(true);
         }
 
+        if isolation == IsolationType::Tdx {
```

Comment:
Can we implement this everywhere instead of just in TDX?

(Meta concern: I don't really like that this crate essentially knows the capabilities of the various virt_* users. We should move this to a builder pattern and require the virt_* implementations to specify the appropriate flags based on their capabilities.)

Reply:
Added for SNP too